docs/training.md

```diff
@@ -1,17 +1,21 @@
 # Training
 
+## Videos
+
+We also have a YouTube playlist of videos that includes workshops you can follow along with:
+
+[![Videos Screenshot](assets/videos.png) Open the playlist](https://youtube.com/playlist?list=PLGHfqDpnXFXLHfeapfvtt9URtUF1geuBo)
+
 ## Hands-On
 
-We've created a Katacoda course featuring beginner and intermediate lessons. These allow to you try out Argo Workflows
-in your web browser without needing to install anything on your computer. Each lesson starts up a Kubernetes cluster
-that you can access via a web browser.
+We are working on a new web-based course featuring beginner and intermediate lessons. These allow to you try out Argo Workflows in your web browser without needing to install anything on your computer. Each lesson starts up a Kubernetes cluster that you can access via a web browser.
 
-The course will take around 2 hours to complete and is the fastest way to learn Argo Workflows.
+The course takes around 2 hours to complete and is the fastest way to learn Argo Workflows.
 
-[![Katacoda Screenshot](assets/katacoda.png) Open the course](https://www.katacoda.com/argoproj/courses/argo-workflows/)
+![Katacoda Screenshot](assets/katacoda.png)
 
-## Videos
+We expect the new course to be live by Q3 '22. Please upvote or comment on [this issue](https://github.com/argoproj/argo-workflows/issues/8899) with your support and feedback for the hands-on course.
```

Comment:
lets not make any promises eh?

Reply:
ha, I'm pretty confident I can make it happen, but yes, happy to not set expectations too high 👍